src/protocols/LoRaWAN/LoRaWAN.h

```diff
@@ -545,47 +559,43 @@ class LoRaWANNode {
     int16_t setBufferSession(uint8_t* persistentBuffer);
 
     /*!
-      \brief Restore session by loading information from persistent storage.
-      \returns \ref status_codes
-    */
-    int16_t restore(uint16_t checkSum, uint16_t lwMode, uint8_t lwClass, uint8_t freqPlan);
-
-    /*!
-      \brief Join network by performing over-the-air activation. By this procedure,
-      the device will perform an exchange with the network server and set all necessary configuration. 
+      \brief Set the device credentials and activation configuration
       \param joinEUI 8-byte application identifier.
       \param devEUI 8-byte device identifier.
       \param nwkKey Pointer to the network AES-128 key.
       \param appKey Pointer to the application AES-128 key.
-      \param force Set to true to force joining even if previously joined.
+    */
+    void beginOTAA(uint64_t joinEUI, uint64_t devEUI, uint8_t* nwkKey, uint8_t* appKey);
+
```

Comment:
Since `begin` methods are now `void`, and new `activate` methods were added, we need to update the examples to reflect that.

Reply:
A product of the testing!